packages/web-components/src/components/rux-modal/rux-modal.tsx

```diff
@@ -73,26 +92,27 @@ export class RuxModal {
     handleClick(ev: MouseEvent) {
         const wrapper = this._getWrapper()
         if (ev.composedPath()[0] === wrapper) {
-            this.ruxModalClosed.emit(false)
+            // this.ruxModalClosed.emit(false)
             this.open = false
         }
     }
 
     @Watch('open')
-    validateName(isOpen: boolean) {
-        if (isOpen) {
+    handleOpen(isOpen: boolean) {
```

Comment:
Changed to handleOpen because I have no idea what name would be validated by this